ehsm-service/index.js

```diff
@@ -0,0 +1,167 @@
+const express = require('express')
+const ffi = require('ffi-napi');
+const app = express();
+// app.use(express.urlencoded());
+app.use(express.json());
+const logger = require('./logs/logger');
+
+const pramas = require('./constans.js')
+const port = 3000;
+
+const result = (code, msg, data) => {
+  return {
+    code: code,
+    message: msg,
+    result: {
+      data
+    }
+  }
+}
+const apis= {
+  CreateKey: 'CreateKey',
+  Encrypt: 'Encrypt',
+  Decrypt: 'Decrypt',
+  GenerateDataKey: 'GenerateDataKey',
+  GenerateDataKeyWithoutPlaintext: 'GenerateDataKeyWithoutPlaintext',
+  ExportDataKey: 'ExportDataKey',
+  Sign: 'Sign',
+  Verify: 'Verify',
+}
+
+/**
+ * 校验sign
+ */
+const _checkSign = function (req,res,next){
+  const _logData = {
+    body:req.body,
+    query:req.query
+  }
+  logger.info(JSON.stringify(_logData));
+  const {appid,nonce,timestamp,sign, payload} = req.body;
+  if(!appid || !nonce|| !timestamp  || !sign || !payload) {
+    res.send(result(401,'缺少必要参数'));
+    return;
+  }
+  // todo sign 校验
+  next();
+}
+/**
+ * 校验payload
+ */
+ const _checkPayload = function (req,res,next){
+   const action = req.query.Action;
+  const { payload } = req.body;
+  const currentPayLoad = pramas[action];
+  for (const key in currentPayLoad) {
+    if(payload[key] == undefined) {
+      res.send(result(401,'payload参数不全'));
```

Comment:
Change all wordings to English, including all /*comments*/.

Reply:
ok